### PR TITLE
feat(mathml): always load MathML font, update Polyfill, use latest woff2 font

### PR DESCRIPTION
**Pull request by caugner**
## Summary

Fixes https://github.com/mdn/yari/issues/7607.

### Problem

For browsers that don't support MathML, we load a MathML polyfill and a custom font ([STIX TwoMath](https://github.com/stipub/stixfonts)). This means that other browsers that support MathML use their default font for MathML, which might be different from that font.

### Solution

Continue to load a MathML polyfill for browsers without support, but _always_ load the custom font.

**Note**: The Polyfill doesn't apply to live samples, which are rendered in an (independent) iframe, but https://github.com/mdn/yari/pull/7621 fixes that.

---

## Screenshots

_No visual change observed._

### After

---

## How did you test this change?

Opened these pages in Chrome, Firefox and Safari:
- http://localhost:3000/en-US/docs/Web/MathML/Examples/MathML_Pythagorean_Theorem
- http://localhost:3000/en-US/docs/Learn/MathML/First_steps/Getting_started#the_display_attribute (Note: This is a live sample, to which the polyfill doesn't apply.)

### CODE REVIEW

File: client/src/document/mathml-polyfill/mathml-font.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@font-face {
+  font-family: "STIXRegular";
+  font-weight: normal;
+  src: local("STIXTwoText") url("./font/STIXTwoMath-Regular.woff2")
```
Comment (caugner) [inline client/src/document/mathml-polyfill/mathml-font.scss:4]:
```suggestion
  src: local("STIXTwoMath-Regular") url("./font/STIXTwoMath-Regular.woff2")
```

File: client/src/document/mathml-polyfill/mathml-font.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@font-face {
+  font-family: "STIXRegular";
+  font-weight: normal;
+  src: local("STIXTwoMath-Regular") url("./font/STIXTwoMath-Regular.woff2")
+    format("woff2");
+}
+
+math {
+  font-family: "STIXRegular";
```
Comment (caugner) [inline client/src/document/mathml-polyfill/mathml-font.scss:9]:
```suggestion
  font-family: "STIXTwoMath-Regular";
  font-weight: normal;
  src: local("STIXTwoMath-Regular") url("./font/STIXTwoMath-Regular.woff2")
    format("woff2");
}

math {
  font-family: "STIXTwoMath-Regular";
```

File: client/src/document/mathml-polyfill/mathml.css [hunk 3]
```diff
@@ -186,7 +214,7 @@ ms[rquote]:after {
 [mathvariant="bold-italic"],
 [mathvariant="bold-sans-serif"],
 [mathvariant="sans-serif-bold-italic"] {
-  font-weight: var(--font-body-strong-weight);
+  font-weight: bold;
```
Comment (caugner) [inline client/src/document/mathml-polyfill/mathml.css:217]:
```suggestion
  font-weight: var(--font-body-strong-weight);
```

**Review by fiji-flo — APPROVED** [review]
Looks great.

**Pull request merged**